detection_rules/schemas/v7_8.py

```diff
@@ -70,8 +70,8 @@ class ThreatTactic(jsl.Document):
         reference = jsl.StringField(MITRE_URL_PATTERN.format(type='tactics'))
 
     class ThreatTechnique(jsl.Document):
-        id = jsl.StringField(enum=list(technique_lookup))
-        name = jsl.StringField(enum=TECHNIQUES)
+        id = jsl.StringField(enum=sorted(list(technique_lookup)))
+        name = jsl.StringField()
```

Comment:
I removed this enum here because one of the new unit test provides much more informative information then just throwing up a massive validation error dump

Reply:
++++